### PR TITLE
S3 to S3 scenario

### PR DESCRIPTION
**Pull request by tplevko**

### CODE REVIEW

File: rest-tests/src/test/resources/features/common.feature [hunk 1]
```diff
@@ -32,3 +32,18 @@ Feature: Common scenarios
     Then creates SF lead with first name: "John", last name: "Doe", email: "jdoe@acme.com" and company: "ACME"
     Then validates SF to DB created new lead with first name: "John", last name: "Doe", email: "jdoe@acme.com"
 	Then cleans after SF to DB, removes user with first name: "John" and last name: "Doe"
+
+  @integrations-s3-to-s3
+  Scenario: S3 - S3 integration
+    Given cleans S3 to S3 scenario, removes two sample buckets with names: "syndesis-bucket-in, syndesis-bucket-out"
```
Comment (dsimansk) [inline rest-tests/src/test/resources/features/common.feature]: - It doesn't seem like 3rd person, could we omit the `s` from all the verbs? :)

- And pls put it to separate feature file named like `integration-s3-to-s3`.

File: utilities/src/main/java/io/syndesis/qe/utils/S3Utils.java [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+	public String readTextFileContentFromBucket(String bucketName, String fileName) {
+
+		final S3Object s3object = s3client.getObject(bucketName, fileName);
+		final S3ObjectInputStream inputStream = s3object.getObjectContent();
+
+		final StringWriter writer = new StringWriter();
+		try {
+			IOUtils.copy(inputStream, writer, "UTF-8");
+		} catch (IOException ex) {
+			Logger.getLogger(S3Utils.class.getName()).log(Level.SEVERE, null, ex);
```
Comment (dsimansk) [inline utilities/src/main/java/io/syndesis/qe/utils/S3Utils.java]: lombok's `log`?

File: utilities/src/main/java/io/syndesis/qe/validation/TwSfValidationSteps.java [hunk 3]
```diff
@@ -95,13 +92,13 @@ public void validateIntegration(String record) throws TwitterException {
...
-		assertThat("Contact has appeard in salesforce", contactCreated, is(true));
+		Assertions.assertThat(contactCreated).as("Contact has appeard in salesforce").isEqualTo(true);
 		log.info("Contact appeared in salesforce. It took {}s to create contact.", TimeUnit.MILLISECONDS.toSeconds(System.currentTimeMillis() - start));
 
 		final Contact createdContact = getSalesforceContact(salesforce, accountsDirectory.getAccount(RestConstants.getInstance().getSYNDESIS_TALKY_ACCOUNT()).get().getProperty("screenName")).get();
-		assertThat(createdContact.getDescription(), Matchers.startsWith(record));
-		assertThat(createdContact.getFirstName(), not(isEmptyString()));
-		assertThat(createdContact.getLastname(), not(isEmptyString()));
+		Assertions.assertThat(createdContact.getDescription()).startsWith(record);
+		Assertions.assertThat(createdContact.getFirstName()).isEmpty();
```
Comment (dsimansk) [inline utilities/src/main/java/io/syndesis/qe/validation/TwSfValidationSteps.java]: `FirstName` and `LastName` should be asserted as `notEmpty`.

File: utilities/pom.xml [hunk 4]
```diff
@@ -96,10 +96,21 @@
...
+
 		<!-- Third party services dependencies -->
 		<dependency>
 			<groupId>org.twitter4j</groupId>
 			<artifactId>twitter4j-core</artifactId>
+		</dependency>		
+		<dependency>
+			<groupId>com.amazonaws</groupId>
+			<artifactId>aws-java-sdk</artifactId>
+			<version>1.11.257</version>
```
Comment (dsimansk) [inline utilities/pom.xml]: versions as properties and to dependencyManagement section of root's pom

**Comment by tplevko** [comment]
Comments reflected

File: utilities/pom.xml [hunk 5]
```diff
@@ -96,10 +96,20 @@
 			<groupId>com.google.guava</groupId>
 			<artifactId>guava</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.jboss.resteasy</groupId>
+			<artifactId>resteasy-jackson-provider</artifactId>
+			<version>3.1.2.Final</version>
```
Comment (dsimansk) [inline utilities/pom.xml]: Pls this version also.

Comment (tplevko) [reply utilities/pom.xml]: fixed

**Pull request merged**